docs/xbcloud-binary-fifo-datasink.md

```diff
@@ -0,0 +1,56 @@
+## FIFO datasink
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements a new datasink that uses `FIFO` (named pipes) to make the backup process faster by streaming data in parallel from XtraBackup to `xbcloud`. 
+
+When taking a backup, you can either save the backup files locally on the same server, stream the backup to different servers, or to object storage. Percona XtraBackup already supports streaming backups using [`xbcloud`](xbcloud-binary-overview.md). 
```

Comment:
When taking a backup, you can save the files locally or stream the files to either different servers or object storage.